private-templates-service/client/src/components/AdaptiveCardPanel/AdaptiveCardPanel.tsx

```diff
@@ -1,20 +1,21 @@
 import * as React from "react";
 import { Container, TemplateName, ACWrapper } from "./styled";
 import AdaptiveCard from '../Common/AdaptiveCard';
-import { renderAdaptiveCard } from "../Common/AdaptiveCard/AdaptiveCard";
 
 interface Props {
   toggleModal: () => void;
+  template: any
 }
 
 class AdaptiveCardPanel extends React.Component<Props> {
   render() {
+    let temp = this.props.template;
     return (
       <Container>
         <ACWrapper>
-          <AdaptiveCard />
+          <AdaptiveCard toggleModal={this.props.toggleModal} cardtemplate={temp} />
         </ACWrapper>
-        <TemplateName onClick={this.props.toggleModal}>Template Name</TemplateName>
+        <TemplateName onClick={this.props.toggleModal}>{this.props.template.name}</TemplateName>
```

Comment:
Either make this use temp, or make line 16 just use this.props.template

Reply:
Done.